src/tools/rbd/action/MergeDiff.cc

```diff
@@ -196,7 +196,7 @@ static int do_merge_diff(const char *first, const char *second,
     }
   }
 
-  //We just handle the case like 'banner, [ftag], [ttag], stag, [wztag]*,etag',
+  //We just handle the case like 'banner, [ftag], [ttag], [stag], [wztag]*, [etag]',
```

Comment:
I think this was an attempt to say that the "s" and "e" tags are not optional and the ones in the brackets are optional.